### ISSUE TITLE
Low Accuracy than Caffe

### ISSUE THREAD

**Issue by mahfuj9346449** [kratzert/finetune_alexnet_with_tensorflow#13]
I fine-tuned using caffe and I got 61% accuracy whereas by using this code in tensorflow I got 56% accuracy. For both cases I trained only fc8 layer. Can you explain why it is?

**Comment by kratzert** [comment]
Hmm interesting, I didn't oberserve similar problems. Of course accuracy might not be the exact same, but should be somehow comparable. I don't have caffe installed anymore, and if you don't post your full specs for your caffe training setup it's hard to tell where the problem might be. Maybe different initialization? Different learning rate? Different optimizer? There are so much variables that your two sentences question does only give space for speculations

**Comment by mahfuj9346449** [comment]
Thank you for your answer.

**Comment by mahfuj9346449** [comment]
One thing can be noted, in caffe I used the learning rate 10 times in fc8 than other layers. Is there any option to use different learning rate in different layers in tensorflow?

**Comment by kratzert** [comment]
there is, see this link:
https://stackoverflow.com/questions/34945554/how-to-set-layer-wise-learning-rate-in-tensorflow

But if I understood you correctly, you are only training the last layer. So why not setting the global learning rate to the value you want? You don't need layer-wise learning rate at all for this?!

**Comment by mahfuj9346449** [comment]
I got it.
